segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/chunklayer/ChunkStorageMetrics.java

```diff
@@ -94,4 +94,8 @@ public class ChunkStorageMetrics {
     static final Counter SLTS_SYSTEM_TRUNCATE_COUNT = STATS_LOGGER.createCounter(MetricsNames.SLTS_SYSTEM_TRUNCATE_COUNT);
 
     static final Counter LARGE_CONCAT_COUNT = STATS_LOGGER.createCounter(MetricsNames.STORAGE_LARGE_CONCAT_COUNT);
+
+    static final OpStatsLogger SLTS_HEALTH_PARALLEL_THROTTLE = STATS_LOGGER.createStats(MetricsNames.SLTS_HEALTH_PARALLEL_THROTTLE);
```

Comment:
Would we be showing this on our Grafana throttle dashboard alongside other throttles?

Reply:
Yes, that is the intention.